### PR TITLE
new file

### PR DESCRIPTION
**Pull request by miphimarketing**
Hi check my new files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<head>
+	<title>Gradient Background</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body id="gradient">
+	<h1>Background Generator</h1>
+	<input class="color1" type="color" name="color1" value="#00ff00">
+	<input class="color2" type="color" name="color2" value="#ff0000">
+	<h2>Current CSS Background</h2>
+	<h3></h3>
```
Comment (digimaxasia) [inline index.html:12]: This should be <h2>

File: index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Gradient Background</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
```
Comment (digimaxasia) [inline index.html:6]: This is not head

**Review by digimaxasia — COMMENTED** [review]
Good job

File: index.html (hunk 1 above)
Comment (miphimarketing) [reply index.html:12]: I have fixed

**Pull request merged**